### PR TITLE
Add circle ci

### PR DESCRIPTION
**Pull request by piyushtiw**

### CODE REVIEW

**Comment by piyushtiw** [comment]
@vipulnsward _a

File: Gemfile [hunk 1]
```diff
@@ -52,6 +52,15 @@ gem "active_model_serializers", "~> 0.9.4"
...
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  # For splitting tests across CircleCI containers
+  gem "knapsack"
+  # customizable MiniTest output formats
+  gem "minitest-reporters", require: false
+  # Minitest reporter plugin for CircleCI. Gerates JUnit xml reports from tests. https://github.com/circleci/minitest-ci
+  gem "minitest-ci"
+  gem "minitest", "5.10.3"
+  # Call "byebug" anywhere in the code to stop execution and get a debugger console
+  gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
```
Comment (vipulnsward) [inline Gemfile]: Please remove, already defined above

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            - vendor/node_modules
+
+      - run: mkdir reports
+
+     # Database setup
+      - run: cp config/database.yml.ci config/database.yml
+      - run: bundle exec rake db:create db:schema:load --trace
+      # Enable if using yarn
+      - run: bin/yarn install
+      - run: bin/webpack
```
Comment (vipulnsward) [inline .circleci/config.yml]: Please add elm package install as well. This is not going to work

**Comment by vipulnsward** [comment]
Fix conflicts

**Comment by piyushtiw** [comment]
@vipulnsward _a

**Pull request merged**